tablo/csv_utils.py

```diff
@@ -32,109 +29,141 @@
     ('x_', 'y_')
 )
 
+DATE_FORMATS = (
```

Comment:
This will seem to make date parsing much more flexible, but isn't it possible that a date string may conform to more than one? For instance "02-03-04" (assuming `%y` is 2-digit date).

Reply:
It's possible, but it's just for determining whether a column is date or not. The actual parsing is done by `pandas`, and it should (hopefully!) handle such cases internally.